# HG changeset patch
# User Armin Müller
# Date 1424686727 -3600
#      Mon Feb 23 11:18:47 2015 +0100
# Node ID 290adbbe95bde8a484f2328e11baa94e70937701
# Parent  13bc2737177ca6d072b6a61e2a750c8bb839635b
libgui/languages/de_DE.ts: Fix typos in german translation

reviewed and pushed by Andreas Weber

diff --git a/libgui/languages/de_DE.ts b/libgui/languages/de_DE.ts
--- a/libgui/languages/de_DE.ts
+++ b/libgui/languages/de_DE.ts
@@ -818,17 +818,17 @@ Do you want to choose another name?</sou
     <message>
         <location line="-776"/>
         <source>Show Octave directory</source>
         <translation>Aktuelles Octave Verzeichnis anzeigen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Go to current Octave directory</source>
-        <translation>Zum aktuellen Octave Verezichnis gehen</translation>
+        <translation>Zum aktuellen Octave Verzeichnis gehen</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Set Octave directory</source>
         <translation>Octave Verzeichnis setzen</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -1138,17 +1138,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+2"/>
         <source>Search from start</source>
         <translation>Suche vom Beginn</translation>
     </message>
     <message>
         <location line="+199"/>
         <source>Replace Result</source>
-        <translation>Ergebins der Ersetzungen</translation>
+        <translation>Ergebnis der Ersetzungen</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>%1 items replaced</source>
         <translation>%1 Vorkommnisse ersetzt</translation>
     </message>
     <message>
         <location line="+15"/>
@@ -1665,17 +1665,17 @@ instead?</source>
         <location line="-1129"/>
         <location line="+1132"/>
         <source>Browse directories</source>
         <translation>Verzeichnis suchen</translation>
     </message>
     <message>
         <location line="-175"/>
         <source>Quit Debug Mode</source>
-        <translation type="unfinished">Debug Modus beenden</translation>
+        <translation type="unfinished">Debug-Modus beenden</translation>
     </message>
     <message>
         <location line="+38"/>
         <source>&amp;Window</source>
         <translation>&amp;Fenster</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -2377,17 +2377,17 @@ Die Octave Benutzeroberfläche muss jetzt geschlossen werden.</translation>
     <message>
         <location line="-48"/>
         <source>Language (requires restart)</source>
         <translation>Sprache (Neustart erforderlich)</translation>
     </message>
     <message>
         <location line="-50"/>
         <source>Icon size</source>
-        <translation>Icongröße</translation>
+        <translation>Icon-Größe</translation>
     </message>
     <message>
         <location line="+1589"/>
         <source>Synchronize Octave working directory with file browser</source>
         <translation>Octave Arbeitsverzeichnis und Dateibrowser synchronisieren</translation>
     </message>
     <message>
         <location line="+125"/>
@@ -3232,17 +3232,17 @@ Do you want to use the shortcut anyhow r
         <translation>Befehlsfenster</translation>
     </message>
 </context>
 <context>
     <name>webinfo</name>
     <message>
         <location filename="../src/qtinfo/webinfo.cc" line="+82"/>
         <source>Type here and press &apos;Return&apos; to search</source>
-        <translation>Suchbegriff eingeben und mit &apos;Enter&apos; die Suche starten</translation>
+        <translation>Suchbegriff eingeben und mit der &apos;Eingabetaste&apos; die Suche starten</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Global search</source>
         <translation>Globale Suche</translation>
     </message>
     <message>
         <location line="+18"/>
